### PR TITLE
feat add appeal

### PR DESCRIPTION
**Pull request by n1c01a5**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 REACT_APP_STORE_PROVIDER=https://develop.kleros.in
 REACT_APP_PROD_ARBITRATOR_ADDRESS=0x6183694b0f853dff99cde9ad322973ac580b2127
-REACT_APP_DEV_ARBITRATOR_ADDRESS=0x7a5a9d8235a7bd3cd75adc8f1089c51da113b2cc
+REACT_APP_DEV_ARBITRATOR_ADDRESS=0x6183694b0f853dff99cde9ad322973ac580b2127
```
Comment (satello) [inline .env:3]: You want the other address `0x7a5a9d8235a7bd3cd75adc8f1089c51da113b2cc` as DEV. Can you switch it back?

File: src/containers/contract/index.js [hunk 2]
```diff
@@ -113,10 +99,12 @@ class Contract extends PureComponent {
     )
   }
 
-  showEmptyContractEl = contract => contract.data.amount.e === 0
+  showEmptyContractEl = contract =>
```
Comment (satello) [inline src/containers/contract/index.js:102]: What is El stand for?

File: src/sagas/contract.js [hunk 3]
```diff
@@ -97,7 +98,40 @@ function* fetchContract({ payload: { contractAddress } }) {
...
+      )
+    } catch (err) {
+      console.log(err)
+      throw new Error('Error current ruling failed')
+    }
+  }
+
+  let currentSession
+
+  try {
```
Comment (satello) [inline src/sagas/contract.js:118]: Why not do the calls in parallel and try {} catch {} them all in the same block. So like 

```[rulingData, currentSession] = yield all([call(kleros.arbitrator.currentRulingForDispute, ...), call(kleros.arbitrator.getSession)])```

**Review by satello — COMMENTED** [review]
Where is the createAppeal part in the Saga?

File: src/containers/contract/index.js (hunk 2 above)
Comment (n1c01a5) [reply src/containers/contract/index.js:102]: I have a bug without `contract.data.amount.e === 0` but I can't remember which one

File: src/sagas/contract.js (hunk 3 above)
Comment (n1c01a5) [reply src/sagas/contract.js:118]: yep

**Pull request merged**